tests/diffusion_dislocation.prm

```diff
@@ -0,0 +1,109 @@
+set Dimension = 2
```

Comment:
Thanks for adding a test. You need to add some of the output files as well, though.